pytradfri/gateway.py

```diff
@@ -287,7 +287,8 @@ def update(self):
         def process_result(result):
             self.raw = result
 
-        return Command("get", self.path, process_result=process_result)
+        cmd = Command("get", self.path, process_result=process_result)
+        return cmd if self.raw else None
```

Comment:
The caller of this method expects a Command instance. We can't just change the type. If we don't want to return a Command here it's probably better to raise a custom exception and require the caller to handle that

Reply:
I did change all places inside the library where it is used, but of course there can be / are external usage as well.

I have now added a new custom exception, and changed smart_task (setting of dimmer timer) to handle the exception.

A short life test did not show problems.